addons/l10n_pe_website_sale/views/templates.xml

```diff
@@ -38,7 +38,7 @@
 
         <!-- show city -->
         <div t-attf-class="mb-3 #{error.get('city_id') and 'o_has_error' or ''} col-lg-6 div_city_id"
-            t-attf-style="#{(country and country.code != 'PE') and 'd-none' or ''}">
+            t-att-style="(country and country.code != 'PE') and 'display: none'">
```

Comment:
`display: none` change seems good but could we keep the t-attf-style? if it's just because it was a little bit unnecessary, since we are in stable and people can customize template, changing it is slightly less stable.

but the possibility that this cause an issue should be rather low (it's not a good idea to do XPATH relying on style/t-attf-style/t-att-style since they have a high probability of being modified)